File: VivonsExpos/php/utilisateur.php
<?php
class Utilisateur{
    private $login;
    private $mdp;
    private $nom;
    private $prenom;
    private $telephone;
    private $mail;
    private $typeUtilisateur;
    private $raisonSociale;
    private $activite;
    private $anneeInscription;
    private $siteInternet;
    private $exposer;
    private $codeS;

    public function getLogin(){
        return $this->login;
    }
    public function setLogin($login){
        $this->login=$login
    }
    public function getMdp(){
        return $this->mdp
    }
    public function setMdp($mdp){
        $this->mdp=$mdp;
    }
    public function getNom(){
        return $this->nom;
    }
    public function setNom($nom){
        $this->nom=$nom;
    }
    public function getPrenom(){
        return $this->prenom;
    }
    public function setPrenom($prenom){
        $this->prenom=$prenom;
    }
    public function getTelephone(){
        return $this->telephone;
    }
    public function setTelephone($telephone){
        $this->telephone=$telephone;
    }
    public function getMail(){
        return $this->mail;
    }
    public function setMail(){
        $this->mail=$mail;
    }
    public function getTypeUtilisateur(){
        return $this->typeUtilisateur;
    }
    public function setTypeUtilisateur($typeUtilisateur){
        $this->typeUtilisateur=$typeUtilisateur;
    }
    public function getRaisonSociale(){
        return $this->raisonSociale;
    }
    public function setRaisonSociale($raisonSociale){
        $this->raisonSociale=$raisonSociale;
    }
    public function getActivite(){
        return $this->activite;
    }
    public function setActivite($activite){
        $this->activite=$activite;
    }
    public function getAnneeInscription(){
        return $this->anneeInscription;
    }
    public function setAnneeInscription($anneeInscription){
        $this->anneeInscription=$anneeInscription;
    }
    public function getSiteInternet(){
        return $this->siteInternet;
    }
    public function setSiteInternet($siteInternet){
        $this->siteInternet=$siteInternet;
    }
    public function getExposer(){
        return $this->exposer;
    }
    public function setExposer($exposer){
        $this->exposer=$exposer;
    }
    public function getCodeS(){
        return $this->codeS;
    }
    public function setCodeS($codeS){
        $this->codeS=$codeS;
    }


}

File: VivonsExpos/app/src/main/java/com/example/vivonsexpos/DTO/Utilisateur.java
package com.example.vivonsexpos.DTO;

public class Utilisateur {
    private String login;
    private String mdp;
    private String nom;
    private String prenom;
    private String telephone;
    private String mail;
    private String typeUtilisateur;
    private String raisonSociale;
    private String activite;
    private String anneeInscription;
    private String siteInternet;
    private  boolean exposer;
    private String codeS;
    private boolean InscriptionValider;


    public Utilisateur(String pLogin, String pmdp, String pnom, String pprenom, String ptelephone, String pmail, String pTypeUtilisateur, String pRaisonSocial, String pActivite,  String pAnneeInscription, String pSiteInternet,  String pCodeS, boolean pExposer, boolean pInscriptionValider){
        this.login =  pLogin;
        this.mdp = pmdp;
        this.nom = pnom;
        this.prenom= pprenom;
        this.telephone = ptelephone;
        this.mail = pmail;
        this.typeUtilisateur = pTypeUtilisateur;
        this.raisonSociale = pRaisonSocial;
        this.activite = pActivite;
        this.anneeInscription = pAnneeInscription;
        this.siteInternet = pSiteInternet;
        this.codeS = pCodeS;
        this.exposer = pExposer;
        this.InscriptionValider = pInscriptionValider;
    }



}
